src/hooks/useIsProposalOwner.ts

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react'
+
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+
+import { ProposalAttributes } from '../entities/Proposal/types'
+
+export default function useIsProposalOwner(proposal?: ProposalAttributes | null) {
+  const [account] = useAuthContext()
+  const isOwner = useMemo(() => !!(proposal && account && proposal.user === account), [proposal, account])
```

Comment:
Maybe this can be a non-hook function that receives both an address and a proposal.
And I think at the last comparison we can use isSameAddress(proposal.user, account)

Reply:
Good idea using `isSameAddress(proposal.user, account)`

As for receiving the address, my intention was to avoid passing the address along components, and always getting it from the context if possible. WDYT?